examples/nlp/language_modeling/tuning/megatron_gpt_peft_eval.py

```diff
@@ -119,6 +119,7 @@ def main(cfg) -> None:
         peft_model_cfg.data.test_ds = cfg.model.data.test_ds
         peft_model_cfg.activations_checkpoint_granularity = None
         peft_model_cfg.activations_checkpoint_method = None
+        peft_model_cfg.encoder_seq_length = cfg.model.data.test_ds.max_seq_length
```

Comment:
`encoder_seq_length` is a property of the model, this should ideally be updated during training if the model does support longer contexts.